File: app/src/main/java/com/kieran/winnipegbus/Views/StyledSwipeRefresh.java
package com.kieran.winnipegbus.Views;

import android.content.Context;
import android.support.v4.widget.SwipeRefreshLayout;
import android.util.AttributeSet;

import com.kieran.winnipegbus.R;

public class StyledSwipeRefresh extends SwipeRefreshLayout {
    public StyledSwipeRefresh(Context context) {
        super(context);
        initialize();
    }

    public StyledSwipeRefresh(Context context, AttributeSet attrs) {
        super(context, attrs);
        initialize();
    }

    public void initialize() {
        setColorSchemeResources(R.color.rt_blue, R.color.rt_red);
    }
}

File: app/src/main/java/com/kieran/winnipegbus/views/StyledSwipeRefresh.kt
package com.kieran.winnipegbus.views

import android.content.Context
import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
import android.util.AttributeSet

import com.kieran.winnipegbus.R

class StyledSwipeRefresh : androidx.swiperefreshlayout.widget.SwipeRefreshLayout {
    constructor(context: Context) : super(context) {
        initialize()
    }

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {
        initialize()
    }

    fun initialize() {
        setColorSchemeResources(R.color.rt_blue, R.color.rt_red)
    }
}
